packages/flex-plugin-scripts/src/scripts/start.ts

```diff
@@ -202,17 +204,18 @@ export const start = async (...args: string[]): Promise<StartScript> => {
   }
 
   setCwd(plugin.dir);
-  if (type === WebpackType.Complete || type === WebpackType.JavaScript) {
-    _updatePluginPort(port, plugin.name);
-  }
 
   const options = {
     port,
     type,
     remoteAll: args.includes('--include-remote'),
   };
 
-  return _startDevServer(userInputPlugins, options);
+  if (type !== WebpackType.JavaScript) {
```

Comment:
Move the `let pluginConfig: PluginConfig = {};` to this line before this `if` statement just so that everything is next to each other.